website/docs/docs/deploy/slim-ci-jobs.md

```diff
@@ -1,130 +1,67 @@
-<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/GitLab-Pipeline.png" title="GitLab pull request"/>
-
-### Azure DevOps pull request example
-
-The green checkmark means the dbt builds and tests were successful. Clicking on the dbt Cloud section navigates you to the relevant CI run in dbt Cloud.
-<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/Enabling-CI/ADO CI Check.png" title="Azure DevOps pull request"/>
-
-## Configuring a Slim CI job
-
-Slim CI offers an alternative to running and testing all models in your project, and instead runs and tests only what's necessary based on the changes you've made. Slim CI can decrease the time it takes by running and testing only modified models, which can also reduce unnecessary resource usage on your warehouse/data platform.
-
-A Slim CI job:
-
-- Is triggered by a pull request.
-- Defers to a production job.
-- Includes a command with a `--select state:modified+` selector, which dbt uses to build only new or changed models and their downstream dependents. 
-
-dbt then identifies the models that need to be run and tested using a state comparison to the production job that you've selected.
+    For your dedicated Slim CI job, dbt Labs recommends you set up a separate dbt Cloud environment that's connected to a staging database. This ensures that you have good isolation on where your CI schemas build. Having a separate environment also allows you to set a [custom branch](/faqs/environments/custom-branch-settings) if you want to trigger CI checks off PRs to a pre-release branch.
+1. On your deployment environment page, click **Create One** to create a new job or edit an existing job to open the **Create Job** settings page.
+2. In the **Execution Settings** section: 
+    - For the option **Defer to a previous run state**, choose **Production Job (Production)**. This tells dbt Cloud to compare the manifest of the current job against the project representation that was materialized the last time this job was run successfully. By setting this option, dbt Cloud only checks the modified code and compares the changes against what’s running in production, instead of building the full table or the entire DAG.
+    - For the option **Commands**, enter `dbt build --select state:modified+` in the field. This informs dbt Cloud to build only new or changed models and their downstream dependents. Importantly, state comparison can only happen when there is a deferred job selected to compare state to. For more information, refer to [Deferral and state comparision](/docs/deploy/slim-ci-jobs#deferral-and-state-comparison)
+3. In the **Triggers** section, choose the **Continuous Integration** (CI) tab. Then, enable the **Run on Pull Requests** option. This configures pull requests and new commits to be a trigger for the Slim CI job.
 
 ### Deferral and state comparison  
```

Comment:
I don't think we need this section anymore since it was handled above
.

Reply:
great, thank you! this is very helpful